dagshub/streaming/mount.py

```diff
@@ -20,7 +21,6 @@
               f"Please use install_hooks to access DagsHub hosted files from a python script"
     raise ImportError(err_str)
 
-from fuse import FUSE, FuseOSError, LoggingMixIn, Operations
 
 class DagsHubFUSE(LoggingMixIn, Operations):
     def __init__(self,
```

Comment:
We have to put a #noqa here, because importing from fuse checks for existing libfuse and throws an error if there is none installed.
This is why I'm doing this check above